src/index.js

```diff
@@ -5,8 +5,8 @@ const fp = require("fastify-plugin");
  * by setting the relevant response headers.
  * @param {object} server - Fastify instance.
  */
-async function plugin(server) {
-	server.addHook("onRequest", async (req, res) => {
+async function fastifyDisablecache(server) {
```

Comment:
renamed the function to reflect how fastifyPlugin would name it. 
See:
https://github.com/fastify/fastify-plugin/blob/master/lib/toCamelCase.js